cmd/eskip/eskip.go

```diff
@@ -113,6 +113,18 @@ func main() {
 		exitHint(err)
 	}
 
+	in, out, err = addDefaultMedia(cmd, in, out)
+
+	if err != nil {
+		exitHint(err)
+	}
+
+	writeClient, err := createWriteClient(out)
```

Comment:
What's I'd like to achieve here in the future is the total opposite of what you're saying. I'd like to have the commands only get a read, and write client as parameters. This completely decouples the command by the `medium`. A command doesn't need to know about the `medium`, it only needs a way to read and a way to write routes. And this is represented by the ReadClient and WriteClient.